### PR TITLE
[TECG-297]/[TECG-372]-Last Activity Date on Card: Change the Default

### PR DESCRIPTION
**Pull request by felipeaziz**
order to "Last Activity Date" and adding the "Aplphabetic" order to the
order options combobox.

### CODE REVIEW

File: src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java [hunk 1]
```diff
@@ -166,17 +166,27 @@ private void validateInformations(Technology technology) throws BadRequestExcept
...
       throw new NotFoundException(ValidationMessageEnums.NO_TECHNOLOGY_WAS_FOUND.message());
     } else {
       TechnologiesResponse response = new TechnologiesResponse();
+      sortTechnologiesDefault(techEntities);
       response.setTechnologies(techEntities);
       return response;
     }
   }
 
+  private void sortTechnologiesDefault(List<Technology> techEntities) {
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java]: @felipeaziz Please create a class to sort technologies. Sorting technology is not a TechnologyService's responsability. Doing this, the service implementation will be cleaner.

File: src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java [hunk 2]
```diff
@@ -271,8 +281,35 @@ public Response findTechnologiesByFilter(TechnologyFilter techFilter, User user)
       techFilter.setRecommendationIs("");
     }
     List<Technology> completeList = technologyDAO.findAllActives();
-    List<Technology> dateFilteredList = new ArrayList<>();
+    completeList = filterByLastActivityDate(techFilter, completeList);
+
+    List<Technology> filteredList = new ArrayList<>();
+    if ((techFilter.getTitleContains() == null || techFilter.getTitleContains().isEmpty())
```
Comment (brunocarneiro) [inline src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java]: @felipeaziz  Use StringUtils from apache-commons-utils (something like this :).

Comment (felipeaziz) [inline src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java]: Done

File: src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java (hunk 1 above)
Comment (felipeaziz) [inline src/main/java/com/ciandt/techgallery/service/impl/TechnologyServiceImpl.java]: Move as static method in Technology.class

**Pull request merged**